cpu-features/src/cpu-features.c

```diff
@@ -47,10 +47,8 @@ cpuid(unsigned long leaf,
       unsigned long* edx)
 {
   __asm__("xor %%ecx, %%ecx\n\t"
-          "mov %%ebx,%%edi\n\t"
           "cpuid\n\t"
-          "xchgl %%ebx,%%edi\n\t"
-          : "=a"(*eax), "=D"(*ebx), "=c"(*ecx), "=d"(*edx)
+          : "=a"(*eax), "=b"(*ebx), "=c"(*ecx), "=d"(*edx)
```

Comment:
I think this should be in a separate commit. (As there is already one in the blake PR, it can be removed from this commit.)

Reply:
yeah, that's just here to make it work. It'll go out before merging

Reply:
👍🏻 You can also push it as a separate commit in case the other PR is still not merged.